### ISSUE TITLE
It doesnt work, even when directly accessing swf

### ISSUE THREAD

**Issue by qtqgyt** [krum110487/GameSaver#10]
^
OS: Windows 10
Browser: Opera GX LVL 5 (core: 105.0.4970.76)

**Comment by krum110487** [comment]
Yeah, it doesn't really work anymore now that flash is dead, I would like to re-write it for ruffle. But it would require a whole new rewrite.
